CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
     * `-ingest-storage.read-consistency`: configures the default read consistency.
     * `-ingest-storage.migration.distributor-send-to-ingesters-enabled`: enabled tee-ing writes to classic ingesters and Kafka, used during a live migration to the new ingest storage architecture.
     * `-ingester.partition-ring.*`: configures partitions ring backend.
+* [ENHANCEMENT] Distributor: Add `max_otel_decompressed_recv_msg_size` to limit the otel decompressed write request byte size. #8574
```

Comment:
[Nit] Don't we normally refer to flags rather than YAML parameters in the changelog? For example `-ingest-storage.enabled` just above this changelog entry.

Also, I think the flag should refer to OTLP rather than OTel, since it controls the OTLP endpoint specifically.

```suggestion
* [ENHANCEMENT] Distributor: Add configuration option `-distributor.max-otlp-decompressed-recv-msg-size` to limit otel decompressed write request byte sizes. #8574
```